2011-05-15  Ulrich Drepper  <drepper@gmail.com>

	[BZ #9730]
	* locales/sv_FI (LC_TIME): Fix d_t_fmt and add first_weekday and
	first_workday.
	Patch by Johan Heikkila <johan.heikkila@gmail.com>.

diff a/localedata/locales/sv_FI b/localedata/locales/sv_FI
---
 localedata/locales/sv_FI |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/localedata/locales/sv_FI
+++ b/localedata/locales/sv_FI
@@ -98,7 +98,7 @@
           "<U006E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
           "<U0064><U0065><U0063><U0065><U006D><U0062><U0065><U0072>"
 d_t_fmt  "<U0025><U0061><U0020><U0025><U0065><U002E><U0020><U0025><U0042>/
-<U0074><U0061><U0020><U0025><U0059><U0020><U0025><U0048><U002E><U0025>/
+<U0020><U0025><U0059><U0020><U0025><U0048><U002E><U0025>/
 <U004D><U002E><U0025><U0053>"
 d_fmt    "<U0025><U0064><U002E><U0025><U006D><U002E><U0025><U0059>"
 t_fmt     "<U0025><U0048><U002E><U0025><U004D><U002E><U0025><U0053>"
@@ -107,6 +107,8 @@
 date_fmt       "<U0025><U0061><U0020><U0025><U002D><U0064><U002E><U0025>/
 <U002D><U006D><U002E><U0025><U0059><U0020><U0025><U0048><U002E><U0025>/
 <U004D><U002E><U0025><U0053><U0020><U0025><U007A>"
+first_weekday 2 % Monday
+first_workday 2 % Monday
 END LC_TIME
 
 LC_PAPER
